src/string_bytes.cc

```diff
@@ -449,12 +449,10 @@ Maybe<size_t> StringBytes::StorageSize(Isolate* isolate,
       break;
 
     case BASE64URL:
-      data_size = simdutf::base64_length_from_binary(str->Length(),
-                                                     simdutf::base64_url);
-      break;
-
     case BASE64:
-      data_size = simdutf::base64_length_from_binary(str->Length());
+      data_size = str->Length() % 4 <= 1
+                      ? str->Length() / 4 * 3
+                      : str->Length() / 4 * 3 + (str->Length() % 4) - 1;
```

Comment:
hmm, is this correct? isn't `StorageSize` supposed to work in the opposite direction, i.e. return the size of the base64 string?

Reply:
ah no, StorageSize goes in the same direction as Size, ignore me.